src/components/inputs/localized-money-input/README.md

```diff
@@ -171,3 +196,121 @@ LocalizedMoneyInput.hasEmptyValue([
   { amount: '', currencyCode: 'USD' },
 ]); // ->  true
 ```
+
+### Example
+
+Here's an example of how `LocalizedMoneyInput` would be used inside a form.
+
+```jsx
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { Formik } from 'formik';
+import omitEmpty from 'omit-empty';
+import ErrorMessage from '@commercetools-frontend/ui-kit/messages/error-message';
+import { LocalizedMoneyInput } from '@commercetools-frontend/ui-kit';
+
+
+// the existing document, e.g. from the database
+const doc = {
+  prices: [
+    {
+      currencyCode: 'ALL',
```

Comment:
I would use a different currency in the example as *ALL*  sounds like the english word *all* which might be confusing.